milestone-delivery/deliveries/dappworld_learning_ecosystem-milestone-1.md

```diff
@@ -0,0 +1,14 @@
+# Milestone Delivery :mailbox:
+
+
+**The Ethereum address has been filled out correctly for this milestone and the delivery is according to the official [milestone delivery guidelines](../).**  
+
+* **PR Link:**  [DApp World Learning ecosystem](https://github.com/vechain/grant-program/blob/master/applications/dappworld_learning_ecosystem.md). 
```

Comment:
Link should be https://github.com/vechain/grant-program/pull/120

Reply:
Added correct link in the above merged application file